weatherApp/Features/WeatherList/UI/HomeViewController.swift

```diff
@@ -66,12 +73,22 @@ class HomeViewController: UIViewController {
         
         self.currentWeatherListPresenter.fetchCurrentWeather() { (currentLocation) in
             guard let currentLocation = currentLocation else { return }
+            
             DispatchQueue.main.async {
                 self.currentLocationView.set(currentWeather: currentLocation)
             }
         }
     }
     
+    private func startTimer() {
+        timerObservable = Observable<Int>
```

Comment:
This is one way to reinitialize observable. But I think that it is more clear if you create disposeBag only for this observable. And each time when startTimer is called you can initialize this dispose bag and subscribe to this timer.

Or maybe you can call .dispose() on dispose bag without reinitializing it every time at the start of this func.